VK_App/VK_App/View/NewsViewCell/NewsImageViewCell.swift

```diff
@@ -0,0 +1,20 @@
+// NewsImageViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// ячейка с картинкой
+final class NewsImageViewCell: UITableViewCell, NewsCellConfigurable {
+    // MARK: - Private IBOutlet
+
+    @IBOutlet private var contentImageView: UIImageView!
+
+    // MARK: - Public Methods
+
+    func configure(_ news: News) {
+        guard let photo = news.attachments?.first?.photo?.sizes.last?.url else { return }
+        NetworkService.instance.downloadImage(url: photo) { [weak self] imageData in
```

Comment:
```suggestion
        NetworkService.downloadImage(url: photo) { [weak self] imageData in
```
синглтон не надо в сеть